staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -104,7 +104,7 @@ func (i *informerSync) Check(_ *http.Request) error {
 	// Close stopCh to force checking if informers are synced now.
 	close(stopCh)
 
-	var informersByStarted map[bool][]string
+	var informersByStarted map[bool][]string = make(map[bool][]string)
```

Comment:
If we have to use `make`, how about using the following format:
```suggestion
	informersByStarted := make(map[bool][]string)
```